docs/usage/manual_cleanup.md

```diff
@@ -0,0 +1,105 @@
+
+## Terminology
+
+- __ShootTechnicalID__ : The value of `shoot.status.technicalID` in shoot resource YAML, typically equals to _"shoot--[project-name]--[shoot-name]"_
+- __ShootUid__ : The value of `shoot.status.uid` in shoot resource YAML.
+
+## Kubernetes resource
+As a part of shoot creation, gardener creates some kubernetes resource on Seed and virtual-garden cluster. We have to clean this up so that gardener doesn't waste cpu cycles in reconciling them.
+
+### On virtual-garden cluster:
+- Delete the `shoot.core.gardener.cloud` resource from project namespace.
+- Delete the `backupentry.core.gardener.cloud` resource from project namespace.
+
+### On seed cluster:
+- Delete the namespace with name equivalent to `ShootTechnicalID` from seed cluster on which shoot was hosted.
+    - Remove the finalizers on the namespace scoped resources from that namespace to let namespace deletion succeed and not stuck in terminating state.
+- Delete the associated cluster scoped resource `backupentry.extension.gardener.cloud` with name `ShootTechnicalID--ShootUid` hosted on the referred seed along with the referred `secret` resources.
+
+## Cloud provider resource
+
+As a part of Shoot hosting, gardener create different cloud providers resources like vpc, vm, object in objet store, etc. Missing out on cleaning up some of these resource may end up costing you unnecessary. These resources are spread up across max three accounts one for shoot cloud provide account, dns account, and object store account though usually all might be same for development purpose. Please follow the doc to cleanup all cloud provider resources.
+
+### Shoot cloud provider account
+
+Gardener as a part of shoot reconciliation create the different resource in end-users account. i.e the shoot owners account.
+
+#### AWS
+- Delete the VPC with name `ShootTechnicalID`.
+- Delete the IAM roles related to shoot with name:
+    - `ShootTechnicalID-bastions`
+    - `ShootTechnicalID-nodes`
+
+#### Azure
+- Delete the resource group with the name `ShootTechnicalID-ShootUID[:5]`.
+
+#### GCP
+- Delete the VPC network with the name `ShootTechnicalID`.
+
+#### Openstack
+- Delete the network with the name `ShootTechnicalID` and its dependencies.
+
+
+### Cleaning up Backups
+
+Gardener regularly take the backup of etcd backing shoot cluster.
+While purging shoot one has to consider cleaning up these backups as well.
+You can get the backup provider account
+
+#### AWS
```

Comment:
The instructions for the backup buckets deletion are pretty much the same for all cloud providers. Could generalize them to avoid the duplication. Basically, I like how it is done for the DNS clean up.